source/future.tex

```diff
@@ -51,6 +51,15 @@
 \rSec1[depr.c.headers]{C standard library headers}
 
 \pnum
+For compatibility with the C standard library\indextext{library!C standard},
+the \Cpp standard library provides headers
+\tcode{<ccomplex>}~(\ref{depr.ccomplex.syn}),
```

Comment:
Hm. "Headers" sounds oddly bland, but maybe that's the only accurate description of this part...